src/diffusers/schedulers/scheduling_utils.py

```diff
@@ -36,6 +37,38 @@ class SchedulerOutput(BaseOutput):
     prev_sample: torch.FloatTensor
 
 
+class BaseScheduler(abc.ABC):
```

Comment:
Not super happy about this. If some schedulers don't need a `scale_initial_noise` function we shouldn't force them to call it to get back a no-op, same with `scale_model_input`